src/uct/tcp/tcp_iface.c

```diff
@@ -286,6 +289,8 @@ static UCS_CLASS_INIT_FUNC(uct_tcp_iface_t, uct_md_h md, uct_worker_h worker,
     self->outstanding            = 0;
     self->config.buf_size        = config->super.max_bcopy +
                                    sizeof(uct_tcp_am_hdr_t);
+    self->config.short_size      = config->super.max_short +
```

Comment:
do we want to increase the default for TCP to more than 128?

Reply:
I've done more testing. We can use am_short = am_bcopy - it improves message rates, but I still don't see any performance improvements for latency

Reply:
so let's increase it, even if it's just MR